libkbfs/interfaces.go

```diff
@@ -486,6 +486,16 @@ type KeyMetadata interface {
 		keyGen KeyGen, user keybase1.UID, key CryptPublicKey) (
 		TLFEphemeralPublicKey, EncryptedTLFCryptKeyClientHalf,
 		TLFCryptKeyServerHalfID, bool, error)
+
+	// StoresHistoricTLFCryptKeys returns whether or not historic keys are encrypted
```

Comment:
I think the comment is a bit confusing as written, and it should be more like `whether or not history keys are symmetrically encrypted; if not, they're encrypted per-device.`